File: Server/src/main/java/com/babob/sporcantam/Order/Order.java
package com.babob.sporcantam.Order;

import javax.persistence.*;
import java.time.LocalDate;
import java.util.List;

@Entity
public class Order {
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public List<String> getItems() {
        return items;
    }

    public void setItems(List<String> items) {
        this.items = items;
    }

    public String getCustomerEmail() {
        return customerEmail;
    }

    public void setCustomerEmail(String customerEmail) {
        this.customerEmail = customerEmail;
    }

    public boolean isConfirmed() {
        return confirmed;
    }

    public void setConfirmed(boolean confirmed) {
        this.confirmed = confirmed;
    }

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id", updatable = false, nullable = false)
    private Long id;
    private List<String> items;
    private String customerEmail;
    private boolean confirmed;

}

File: Mobile/Android/app/src/main/java/com/babob/sporcantam/item/Order.kt
package com.babob.sporcantam.item

import java.io.Serializable




class Order: Serializable {

    var isConfirmed:Int
    var customer_email:String
    var order_id: String


    constructor(isConfirmed:Int, customer_email:String, order_id: String) {
        this.isConfirmed = isConfirmed
        this.customer_email = customer_email
        this.order_id = order_id
    }




}